examples/docker-compose/README.md

```diff
@@ -2,6 +2,8 @@
 
 Here is an example on how to reproduce the Nhost stack from a docker-compose file.
 
+Note that the Nhost instance produced will not allow you to do all the things available on Nhost Cloud. Some tabs like Settings may be greyed-out and you will need to configure them using envinoment variables or your own system. Additionally, the [Nhost CLI](https://github.com/nhost/cli) is [unable to be used](https://github.com/nhost/cli/issues/837#issuecomment-1963499492) with self-hosted instances.
```

Comment:
I feel the current wording mischaracterizes a bit the issue and some people may be led to believe we are criplling the self-hosted product, which isn't the case. Maybe a better wording could be:

```suggestion
NOTE: You may notice that some options in the dashboard are greyed-out. Those options are extra services provided by the Nhost Cloud and, thus, not available when self-hosting.
```

Reply:
Could the note about the CLI not being available be kept?

Reply:
I don't think that comment is accurate. The CLI is a development tool. You are free to use it whether you are self-hosting or not. Obviously, you are going to have implement your own deployment pipelines and configuration management system that converts the output of the CLI to your self-hosted instance but there is nothing preventing you from using it.

Reply:
Well I would believe the work involved to get your own system configured to correctly interface with the CLI would be quite the amount of work when nobody has done it before... So I don't think it's not fair to say that the CLI is "compatible".